Library/Homebrew/cmd/commands.rb

```diff
@@ -70,6 +70,6 @@ def find_internal_commands(directory)
       end
     end
 
-    private :internal_commands, :internal_developer_commands, :external_commands, :find_internal_commands
+    # private :internal_commands, :internal_developer_commands, :external_commands, :find_internal_commands
```

Comment:
@alyssais the reason these methods cannot be private is that they are being tested in the ```test/cmd/commands_spec.rb``` file and those tests fail if these methods are private